CMake/cdat_modules/vacumm_deps.cmake

```diff
@@ -0,0 +1 @@
+set(vacumm_deps ${python_pkg} ${numpy_pkg} ${scipy_pkg} ${matplotlib_pkg} ${basemap_pkg} ${configobj_pkg})
```

Comment:
no numpy? no setuptools? if there's a setup.py it probably needs setuptools (distutils)